lib/system/io.nim

```diff
@@ -666,7 +666,7 @@ proc readFile*(filename: string): TaintedString {.tags: [ReadIOEffect], benign.}
   var f: File
   if open(f, filename):
     try:
-      result = readAll(f).TaintedString
```

Comment:
Unrelated commit that could cause harm.

Reply:
I don't think so. `readAll` already returns a TaintedString.

Reply:
Exactly what @Clyybber says. Compiler even warns that the original code is a pointless conversion to itself.